app/javascript/components/pages/Home.js

```diff
@@ -1,11 +1,57 @@
-import React from 'react'
-
+import React from "react";
+import { UncontrolledCarousel } from "reactstrap";
 const Home = () => {
   return (
     <>
-      <h1>Welcome to Big Ballerz Apartments!</h1>
+      <div className="greeting">
+        <h1>Welcome to Big Ballerz Condos!</h1>
+      </div>
+      <UncontrolledCarousel
```

Comment:
Love the idea of the carousel! Something to keep in mind, is the intent to have actual data here in the future or just like a preview of what they may see in your app?

Reply:
yes sir